packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -311,6 +300,12 @@ func (m *Manager) uploadWorkflowToS3() {
 	if m.err != nil {
 		return
 	}
+	log.Debug().Msgf("copying %s to temp directory", m.path)
+	err := copyFileRecursivelyToLocation("temp", m.path)
+	if err != nil {
+		m.err = err
+		return
+	}
```

Comment:
qq: is this redundant logic considering in `initializeTempDir` below, there is already a `copyFileRecursivelyToLocation`?

Reply:
Yes, I deleted those lines but haven't pushed changes yet